lib/MusicBrainz/Server/Controller/Release.pm

```diff
@@ -361,6 +361,11 @@ sub add_cover_art : Chained('load') PathPart('add-cover-art') Edit {
         }
     } elsif ($c->form_posted) {
         $c->response->status(500);
+    } elsif (!$c->form_posted && %{ $c->req->query_params }) {
```

Comment:
nitpick, but based on the `elsif ($c->form_posted)` above think `!$c->form_posted` is guaranteed?
```suggestion
    } elsif (%{ $c->req->query_params }) {
```

Reply:
Ah, sorry for missing that. Yep, that makes sense.